components/host-dashboard/HostDashboardExpenses.js

```diff
@@ -220,6 +223,31 @@ const HostDashboardExpenses = ({ hostSlug }) => {
           <HostInfoCard host={data.host} />
         )}
       </Box>
+      {!expenses.loading && data?.host && (
+        <ScheduledExpensesBanner
+          host={data.host}
+          onSubmit={() => {
+            expenses.refetch();
+          }}
+          secondButton={
+            !(query.status === 'SCHEDULED_FOR_PAYMENT' && query.payout === 'BANK_ACCOUNT') ? (
+              <StyledButton
+                buttonSize="tiny"
+                buttonStyle="successSecondary"
+                mr={1}
+                onClick={() => {
+                  router.push({
+                    pathname: `/${hostSlug}/dashboard/expenses`,
+                    query: getQueryParams({ status: 'SCHEDULED_FOR_PAYMENT', payout: 'BANK_ACCOUNT', offset: null }),
+                  });
+                }}
+              >
+                List Expenses
```

Comment:
I18n

If you're using VSCode, this tip helps to quickly add `FormattedMessage`s in the code: https://github.com/opencollective/opencollective/blob/main/team/Benjamin.md#snippet-to-quickly-generate-formattedmessage